Compile with Reeber fails: "no matching function for call to FabComponentBlock"

When I attempt to compile the `LyA` project with Reeber enabled on the Nyx development branch, GCC gives me the following error:

```
../../Util/reeber/src/reeber.cpp:433:98: error: no matching function for call to 
‘FabComponentBlock<double, 3>::FabComponentBlock(diy::GridRef<double, 3>&, 
std::vector<std::__cxx11::basic_string<char> >&, std::vector<diy::GridRef<double, 3>, 
std::allocator<diy::GridRef<double, 3> > >&, int&, int&, const diy::DiscreteBounds&, 
diy::AMRLink::Bounds&, diy::AMRLink::Bounds&, int, AMRLink*&, const Real&, 
const bool&, bool, const Real&)’
  433 |                                 new_link, absolute_rho, negate, /*absolute = */ true, cell_volume),
      |                                                                                                  ^
```

If I use [clangd](https://clangd.llvm.org/) to lint the code, I get the following on line 430:

`No matching constructor for initialization of 'Block' (aka 'FabComponentBlock<double, 3>')`

--- a/Source/IO/Nyx_output.cpp
+++ b/Source/IO/Nyx_output.cpp
@@ -1086,7 +1086,7 @@ Nyx::updateInSitu ()
 #endif
 
 #ifdef REEBER
-    amrex::Vector<Halo>& reeber_halos);
+    amrex::Vector<Halo> reeber_halos;
     halo_find(parent->dtLevel(level), reeber_halos);
     halo_print(reeber_halos);
 #endif

--- a/Util/reeber/src/reeber.cpp
+++ b/Util/reeber/src/reeber.cpp
@@ -430,7 +430,7 @@ std::vector<Halo> compute_halos(diy::mpi::communicator& world,
                         new Block(b->fab, b->extra_names_, b->extra_fabs_, local_ref, local_lev, diy_domain,
                                 l->bounds(),
                                 l->core(), cp.gid(),
-                                new_link, absolute_rho, negate, /*absolute = */ true, cell_volume),
+                                new_link, absolute_rho, negate, /* wrap = */ true, /*absolute = */ true, cell_volume),
                         new_link);
 
                 //if (debug) fmt::print(std::cerr, "Added block gid = {}\n", cp.gid());

